### PR TITLE
Add back container_test

### PR DESCRIPTION
**Pull request by tejal29**
This is to add back container_tests.
I was going to combine it with https://github.com/bazelbuild/rules_docker/pull/239 however the diff was getting bigger.

### CODE REVIEW

File: container/container.bzl [hunk 1]
```diff
@@ -20,6 +20,7 @@ load("//container:import.bzl", "container_import")
 load("//container:load.bzl", "container_load")
 load("//container:pull.bzl", "container_pull")
 load("//container:push.bzl", "container_push")
+load("//contrib:test.bzl", "container_test")
```
Comment (mattmoor) [inline container/container.bzl]: We don't include `contrib` things here.  If it's ready to graduate it should move to `//container`, but I don't think that's the intent of this change?

File: tests/docker/BUILD [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+package(default_visibility = ["//visibility:public"])
+
+load("//container:container.bzl", "container_test")
+
+container_test(
```
Comment (mattmoor) [inline tests/docker/BUILD]: Why do we want the `..../docker/...` path part?

File: container/container.bzl (hunk 1 above)
Comment (tejal29) [reply container/container.bzl]: no.. let me just get it to work here. We have a task internally to convert all image_test.py to use container_test
Maybe then we will ready.. Just created one here https://github.com/bazelbuild/rules_docker/issues/256

File: tests/docker/BUILD (hunk 2 above)
Comment (tejal29) [reply tests/docker/BUILD]: You mean absolute target reference?
I am used to writing full target names. I think its cleaner and easier when you move code around.

**Comment by mattmoor** [comment]
Twiddling for CI

File: container/container.bzl (hunk 1 above)
Comment (dlorenc) [reply container/container.bzl]: Should this line be dropped?

**Comment by tejal29** [comment]
@dlorenc in order to make commit history look pretty i messed up the files. You comment is gone, but the container/BUILD does not load container_test from contrib/test.bzl

**Pull request merged**